@xen-orchestra/web-core/lib/enabled-state/VtsEnabledState.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <UiInfo :accent="state.accent">
+    {{ state.label }}
+  </UiInfo>
+</template>
+
+<script setup lang="ts">
+import UiInfo, { type InfoAccent } from '@core/components/ui/info/UiInfo.vue'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+type StatesMap = Record<'enabled' | 'disabled', { label: string; accent: InfoAccent }>
+
+const { enabled } = defineProps<{
```

Comment:
Perhaps add a default value for ease of use

```suggestion
const { enabled = false } = defineProps<{
}>()
```

Reply:
As we said in our discussion in MM, I think the prop should be required because the _enabled/disabled_ state should always be based on a condition depending on the context.